### DISCUSSION TITLE
Paddle和cuda12有兼容性问题吗?

### DISCUSSION THREAD

**Discussion by zloss** [PaddlePaddle/PaddleOCR#10289] [General]
Paddle_2.5.0+CUDA12.1编译后OCR只能识别到图片下半部分的文字内容，更换成Paddle_2.5.0+CUDA11.8编译就可以识别到整张图片的文字内容，是Paddle和cuda12有兼容性问题吗？（同时也测试了Paddle-2.3.0和cuda12也是同样的问题，只能识别半张图片）
编译环境：
Rocky Linux release 9.2 (Blue Onyx)
gcc (GCC) 11.3.1 20221121 (Red Hat 11.3.1-4)
CUDA-12.1 + CUDNN-8.9.2.26 + NVIDIA-Linux-x86_64-530.30.02  识别半张图片
( CUDA-11.8 + CUDNN-8.9.2.26 +NVIDIA-Linux-x86_64-520.61.05 可以识别整张图片）
显卡是Geforce GTX 1080
代码用的是PaddleOCR/deploy/cpp_infer
编译均能顺利完成，无错误。
测试图片为
![test](https://github.com/PaddlePaddle/PaddleOCR/assets/6358007/ff9b5340-8ace-4b42-8d01-f39abdeb533b)

**Comment by kongxa** [comment]
我用cuda12.6执行ppocr没啥问题 可以正常提取文字
![image](https://github.com/user-attachments/assets/6d0ef65b-9069-499b-80d7-29b911b8c76f)
